ui/apps/platform/webpack.ocp-plugin.config.js

```diff
@@ -15,7 +16,7 @@ const config = {
         chunkFilename: isProd ? '[name]-chunk-[chunkhash].min.js' : '[name]-chunk.js',
     },
     resolve: {
-        extensions: ['.ts', '.tsx', '.js', '.jsx'],
+        extensions: ['.js', '.jsx', '.ts', '.tsx'],
```

Comment:
The ordering here is switch to ensure that a `.js` file that was transpiled from `.ts` file in `node_modules` is preferred when building the dependency graph.